README.md

```diff
@@ -160,14 +198,14 @@ val errorMessage = Command<String>()
 ```
 Show some message in the View:
 ```kotlin
-pm.errorMessage.observable bindTo { message ->
+pm.errorMessage bindTo { message ->
     Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
 }
 ```
 
-When the View is unbound from the PresentationModel, **Command** collects all received values and emits them on binding:
+When the View is pauses, **Command** collects all received values and emits them on resume:
```

Comment:
When the View is paused, Command collects all received values and emits them on resume: